jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/html/AbstractHtmlContainerLoader.java

```diff
@@ -40,5 +49,25 @@ public void loadComponent() {
         componentLoader().loadWhiteSpace(resultComponent, element);
         componentLoader().loadSizeAttributes(resultComponent, element);
         componentLoader().loadThemeList(resultComponent, element);
+
+        String property = element.attributeValue("property");
+        if (!Strings.isNullOrEmpty(property) && resultComponent != null) {
+            Optional<InstanceContainer<?>> instanceContainer = getDataLoaderSupport().loadContainer(element, property);
+            instanceContainer.ifPresent(container -> getHtmlDataBinding().bind(resultComponent, container, property));
+        }
+    }
+
+    protected DataLoaderSupport getDataLoaderSupport() {
+        if (dataLoaderSupport == null) {
+            dataLoaderSupport = applicationContext.getBean(DataLoaderSupport.class, context);
+        }
+        return dataLoaderSupport;
+    }
+
+    protected HtmlContainerReadonlyDataBinding getHtmlDataBinding() {
+        if (htmlComponentReadonlyDataBinding == null) {
+            htmlComponentReadonlyDataBinding = applicationContext.getBean(HtmlContainerReadonlyDataBinding.class, context);
```

Comment:
You don't need to pass the `context` as an additional variable
`DataLoaderSupport` bean requires `context` as a constructor argument

In the `HtmlContainerReadonlyDataBinding` constructor, you are requests for a bean, which will autowired automatically
There's definitely no need for context there

```suggestion
            htmlComponentReadonlyDataBinding = applicationContext.getBean(HtmlContainerReadonlyDataBinding.class);
```